source/sass/cms.scss

```diff
@@ -1,6 +1,26 @@
 // Resets / sensible defaults for CMS-managed content
 
 .cms {
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    position: relative;
+
+    a {
+      text-decoration: inherit;
+      color: inherit;
+      font: inherit;
+
+      &[id] {
+        position: absolute;
+        top: -85px;
```

Comment:
This fixed offset might not work responsively. (try it on 874px wide screen, for example)

Reply:
happy to work in suggestions, is there any way for us to get the "correct" size at specific breakpoints? (just a bunch of media queries would be fine for instance)

Reply:
Two things to consider: 

1.) The `DEV HELP` item makes this worse at more resolutions than it is in reality
2.) The fact that the nav needs to wrap in the first place seems like a bug, possibly a result of a switch to the items in the nav that we made a few months(?) ago.

Between those two, I'd argue that this isn't an issue to solve here, and we should instead file a follow up about the banner itself looking wonky at just above 768px.

Reply:
That sounds quite reasonable: filed https://github.com/mozilla/foundation.mozilla.org/issues/2823

Reply:
@mmmavis this was your objection- do you agree with me?

Reply:
yup yup let's do that in a follow-up